### PR TITLE
Adds new theme [Djelle/milcomarmy]

### PR DESCRIPTION
**Pull request by Djelle**
## Checklist

- [x ] I've read the [publishing documentation](https://hacs.xyz/docs/publish/start).
- [ x] I've added the [HACS action](https://hacs.xyz/docs/publish/action) to my repository.
- [ ] (For integrations only) I've added the [hassfest action](https://developers.home-assistant.io/blog/2020/04/16/hassfest/) to my repository.
- [ x] The actions are passing without any disabled checks in my repository.
- [ x] I've added a link to the action run on my repository below in the links section.
- [ x] I've created a new release of the repository after the validation actions were run successfully.

## Links

Link to current release: <https://github.com/Djelle/milcomarmy/releases/tag/v1.1>
Link to successful HACS action (without the `ignore` key): <https://github.com/Djelle/milcomarmy/actions/runs/7154501817>
Link to successful hassfest action (if integration): <>

### CODE REVIEW

**Comment by Djelle** [comment]
Updated my theme to v2.0. Could not wait for review acceptance :) The submitted v1.1 is still there. Not sure what to do more. I hope my update doesn't mess with my pull request.
Cheers

File: theme [hunk 1]
```diff
@@ -74,5 +74,6 @@
   "th3jesta/ha-lcars",
   "TilmanGriesel/graphite",
   "williamahartman/noctis-solarized",
-  "wowgamr/animated-weather-card"
-]
\ No newline at end of file
+  "wowgamr/animated-weather-card",
+  "Djelle/milcomarmy"
```
Comment (ludeeus) [inline theme:78]: This list needs to be sorted.

Comment (Djelle) [reply theme:78]: Where do you find this code?

Comment (Djelle) [reply theme]: I think I figured out what to do.

File: theme [hunk 2]
```diff
@@ -17,6 +17,7 @@
   "catppuccin/home-assistant",
   "coltondick/nordic-theme-main",
   "DickSwart/swart_ninja_dark_theme",
+  "Djelle/milcomarmy"
```
Comment (tjorim) [inline theme:20]:
```suggestion
  "Djelle/milcomarmy",
```

File: theme [hunk 3]
```diff
@@ -74,5 +75,5 @@
   "th3jesta/ha-lcars",
   "TilmanGriesel/graphite",
   "williamahartman/noctis-solarized",
-  "wowgamr/animated-weather-card"
-]
\ No newline at end of file
+  "wowgamr/animated-weather-card",
```
Comment (tjorim) [inline theme:78]:
```suggestion
  "wowgamr/animated-weather-card"
```

**Review by tjorim — CHANGES_REQUESTED** [review]

**Pull request merged**